### DISCUSSION TITLE
Many attributes forced to "false" can this be changed?

### DISCUSSION THREAD

**Discussion by brentonv** [iBotPeaches/Apktool#2737] [General]
I wanted to use the following attributes as "true" in androidmanifest.xml but they are always forced to "false" in the rebuild

```
<application
    android:usesCleartextTraffic="true"
    android:requestLegacyExternalStorage="true"
```

Is there any way to allow this?

**Comment by iBotPeaches** [comment]
Shouldn't be happening, I'll have to dig.

**Reply by brentonv** [reply]
Thank you @iBotPeaches much appreciated.

Let me know if you want a PR for this as I just figured it was more of a conversation topic and someone might explain what I was missing

**Reply by iBotPeaches** [reply]
I'll always take PRs, but I'm not following how an attribute can be flipped. My only thought is these properties might act like version, which are injected as parameters to aapt and not left in the Manifest file. This then causes the default values to be used since not passed and may be flipped.
